.github/workflows/ci.yaml

```diff
@@ -58,7 +61,7 @@ jobs:
 
     # if you need to skip staging due to flakiness use "-PskipStaging"
     - name: Test sigstore-java
-      run: ./gradlew build
+      run: ./gradlew build -Porg.gradle.java.installations.auto-download=false -PjdkTestVersion=${{ matrix.java-version }}
```

Comment:
@loosebazooka , here's how we execute tests with a given Java version even though we always build with Java 17